src/components/ImageView/index.js

```diff
@@ -129,55 +137,20 @@ class ImageView extends PureComponent {
      * @param {Number} imageHeight
      */
     setImageRegion(imageWidth, imageHeight) {
-        let width = imageWidth;
-        let height = imageHeight;
-        const containerHeight = this.state.containerHeight;
-        const containerWidth = this.state.containerWidth;
-
-        // return if image not loaded yet
-        if (imageHeight <= 0 || containerHeight <= 0) {
+        if (imageHeight <= 0) {
```

Comment:
As I mentioned in the comment below (https://github.com/Expensify/App/pull/8809#discussion_r864069343) , it can sometimes be called last. therefore it is still necessary for the calculations we use below.